robocorp-code/vscode-client/src/locators.ts

```diff
@@ -8,25 +13,41 @@ import { OUTPUT_CHANNEL } from './channel';
 export async function copySelectedToClipboard(locator?: LocatorEntry) {
     let locatorSelected: LocatorEntry | undefined = locator || getSelectedLocator();
     if (locatorSelected) {
-        vscode.env.clipboard.writeText(locatorSelected.name);
+        env.clipboard.writeText(locatorSelected.name);
```

Comment:
Instead of `getSelectedLocator()`, shouldn't it use the one received as a parameter (if available)?

Reply:
This might be just a bit poor choice of variable name for the `locatorSelected` . The previous comment applies here as well

Reply:
You're right, sorry, my bad upon reviewing.